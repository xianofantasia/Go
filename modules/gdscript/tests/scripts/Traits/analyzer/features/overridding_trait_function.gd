trait SomeTrait:
	func some_func(param: Node)

class SomeClass:
	uses SomeTrait

	func some_func(param: Object):
		print("overridden some func")

func test():
	SomeClass.new().some_func(Node.new())
	print("ok")
