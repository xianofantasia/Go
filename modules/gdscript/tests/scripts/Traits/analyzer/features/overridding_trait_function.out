GDTEST_OK
~~ WARNING at line 7: (UNUSED_PARAMETER) The parameter "param" is never used in the function "some_func()". If this is intended, prefix it with an underscore: "_param".
overridden some func
ok
